tools/src/keygen/KeygenOptions.hs

```diff
@@ -32,7 +31,7 @@ secretDataTypeParser = asum
     , CommSecret <$> option auto
         (long "comm-sec" <>
          metavar "N" <>
-         help "Use given committee secret and derive secret key of N-th \
+         help "Use giencodeIntegerven committee secret and derive secret key of N-th \
```

Comment:
You've mistakenly inserted `encodeInteger` in the middle of the other word.

Reply:
:wtf: